qe_metrics/utils/product_utils.py

```diff
@@ -42,7 +42,8 @@ def append_last_updated_arg(query: str, look_back_days: int) -> str:
         str: A query with an argument to filter issues that have been updated in the last 'look_back_days' days
     """
     if any(word in query for word in ["updatedDate", "updated"]):
-        LOGGER.error("Query is already using the 'updatedDate' or 'updated' field. Please remove it:\n{query}")
-        return query
+        raise ValueError(
```

Comment:
based on my previous comment, we should log/slack and continue.

you may have one team with this while another 10 teams are ok and we want to process them.

Reply:
@myakove raising the value error will print the error and continue. I have just changed it to a log message, though. Let me know if that works.